src/extension/background-script/state.ts

```diff
@@ -126,7 +148,13 @@ const state = createState<State>((set, get) => ({
     if (connector) {
       await connector.unload();
     }
-    set({ password: null, connector: null, account: null, nostr: null });
+    set({
```

Comment:
Why do we need that here?

Reply:
because `liquid` is initialized with the `liquidPrivateKey` and can be still used when locked

Reply:
while the extension is locked you mean?

Reply:
there is an instance of the `Liquid`, `Nostr`, `Connector` class memoized in the state. Those also need to be removed from memory here.